pkg/api/datasource_permissions.go

```diff
@@ -0,0 +1,22 @@
+package api
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/models"
+)
+
+type DatasourcePermissionsService interface {
```

Comment:
We usually put the OSS prefix on classes that are going to be override in Enterprise. This way indicates that any change here, affects to Enterprise.

Reply:
But it's an interface, it shouldn't matter, or I'm mistaken? `DatasourcePermissionsServiceImpl` could be `OSSDatasourcePermissionsService` if that's preferred, but the interfaces I'd assume to be under a common name?

Reply:
Oh sorry, I clicked in the wrong line. Yes, it's `DatasourcePermissionsServiceImpl`